test/jdk/java/util/Currency/ValidateISO4217.java

```diff
@@ -177,6 +180,7 @@ static void setUp() throws Exception {
         checkUsage();
         setUpPatchedClasses();
         setUpTestingData();
+        setUpNotYetDefined();
```

Comment:
It may be clearer to move this inside `setUpTestingData()`, and modify the comment there

Reply:
Right, makes more sense that way. Also updated the JBS issues as you requested.